README.md

```diff
@@ -22,16 +22,14 @@ pip install marquez-airflow
 ```python
 from marquez.airflow import MarquezDag as DAG
 from airflow.operators.dummy_operator import DummyOperator
-from datetime import datetime
 
 
 DAG_NAME = 'my_DAG_name'
 
 default_args = {
-    'mqz_namespace': 'namespace_1',
-    'mqz_location': 'github://data-dags/dag_location/',
-    'mqz_input_datasets': ["s3://some_data", "s3://more_data"],
-    'mqz_output_datasets': ["s3://output_data"],
+    'marquez_location': 'github://data-dags/dag_location/',
+    'marquez_input_urns': ["s3://some_data", "s3://more_data"],
+    'marquez_output_urns': ["s3://output_data"],
```

Comment:
_Minor_: We'll want to provide a valid list of URNs when the DatasourceAPI and DatasetAPI are exposed by [`marquez`](https://github.com/MarquezProject/marquez)

Maybe we should open an issue as a reminder? :)

Reply:
yup agree. we could add a validator in the client that can be used by this app?

Reply:
The Marquez API will returning a `400` on attempts to use a URN that's invalid, so my not be worth having the URN validation logic in multiple places?